<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>63</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>minimum value</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="_minSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="_minSpin">
        <property name="suffix">
         <string>/100</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>maximum value</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="_maxSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="_maxSpin">
        <property name="suffix">
         <string>/100</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_minSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>_minSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_maxSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>_maxSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_minSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>_minSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_maxSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>_maxSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
